beps/bep_0011.rst

```diff
@@ -87,9 +87,37 @@ An added contact must not be dropped in the same message.
 
 Except for the initial PEX message the combined amount of added v4/v6 contacts should not exceed 50 entries. The same applies to dropped entries.
 
+    **Implementation Note**: If these limits conflict with large connection pools then implementations can calculate the canonical peer priorities from the perspective of a remote peer to choose which subset of
+connections to prioritize for inclusion in a message.  
+
 Messages must contain at least one of the following fields: ``added, added6, dropped, dropped6``.
 
-Clients may disconnect peers that egregiously violate these constraints.  
+Clients may disconnect peers that egregiously violate these constraints.
+
+
+Filling underpopulated lists
+============================
+
+A combination of the 
+
+* above liveness requirements
+* disconnecting seeds or partial seeds [#BEP-21]_ while seeding
+* disconnecting duplicate peers based on peer IDs between ipv4 and ipv6
+
+can lead to underpopulated ``added`` or ``added6`` lists. This problem frequently arises in swarms dominated by seeds where the seeds have no live connections to propagate and the peers
```

Comment:
a meta comment: I don't think we have any guidelines around this, but I would imagine we would want one of:

* one sentence per line
* lines wrapped at 80 columns

The former makes for better diffs

Reply:
The first point seems fine.
The latter is tedious because you need to reflow sentences with every change.

Reply:
NAK on wrapping at 80 characters for the reason @the8472 stated. Otherwise I'm apathetic towards this. A good diff tool will highlight sub-line changes.